common/bomb.cpp

```diff
@@ -59,6 +58,45 @@ bomb::update()
   return;
 }
 
+std::vector<std::vector<TargetSquare>>
+bomb::identifyTargetSquares()
+{
+  /** Create stacks of squares that can be iterated in a FIFO fashion **/
+  std::vector<std::vector<TargetSquare>> targets;
+  int i = int(mPosition[0]); int j = int(mPosition[1]);
+
+  auto makeSquare = [](int a, int b){
+    return std::make_shared<actor>(a, b, 1, 1, false);
+  };
+
+  // Square bomb is sat on
+  targets.push_back({makeSquare(i, j)});
```

Comment:
Need to init `vec<Square>` as discussed above before `push_back`. There's probably an optimisation to be had by dealing with `*TargetSquare` instead of `TargetSquare` but they are already very lightweight objects.

Reply:
Thankyou for the feedback QA! I've gone ahead and made this change to make it more explicit